FixedBGExamples/KerrComplexScalar/DiagnosticVariables.hpp

```diff
@@ -0,0 +1,26 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef DIAGNOSTICVARIABLES_HPP
+#define DIAGNOSTICVARIABLES_HPP
+
+// assign an enum to each variable
+enum
+{
+    c_chi,  // the conformal factor of the metric, not evolved
+    c_rho,  // the energy density of the SF
```

Comment:
Are we happy with this naming?

Reply:
Will change to be consistent with Boosted example